packages/game-app/src/signup/components/Signup/Signup.tsx

```diff
@@ -46,6 +29,9 @@ const schema = yup.object().shape({
 });
 
 const Signup: React.FC<Props> = () => {
+  const gameRoles = useSelector(dynamicDataSelectors.getGameRoles);
```

Comment:
we can enhance the useRetrieveDevOpsMaturities and useRetrieveGameRoles so that they can also look into the state.

```typescript
const useRetrieveGameRolesInternal = createRequestHook('gameRoles', actions.retrieveGameRoles, {
  errorMessagesScope: '', //TODO
});

function useGameRoles(){
  const request = useRetrieveGameRolesInternal();

  const gameRoles = useSelector(dynamicDataSelectors.getGameRoles)

  return {
    ...request,
    gameRoles
}

```